setup.py

```diff
@@ -285,13 +285,16 @@ def add_dir_name(dir_name, filenames):
     else:
         extra_compile_args.extend(min_extra_compile_args)
 
-    # Added -Wall to get all warnings and -Werror to treat them as errors
-    extra_compile_args.append("-Wall")
-    extra_compile_args.append("-Werror")
+    if os.name == 'nt':
+        extra_compile_args.append("-DBUILDING_DLL")
```

Comment:
Werror doesn't exist for MSVC, and the command already includes "/W3" by default from distutils. but I can put "-Wall" back in sure.